.gitignore

```diff
@@ -12,3 +12,5 @@ docs/
 
 # Dotenv file
 .env
+
+.DS_Store
```

Comment:
Not an issue to have this here, but I recommend that you create a global gitignore file.